### DISCUSSION TITLE
Performance on Github Action

### DISCUSSION THREAD

**Discussion by simone-dandreta** [renovatebot/renovate#22558] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.0.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello,
I run Renovate as a Github Action. I onboarded 10 repositories and each run takes ~2 min and half. As we'll onboard more and more, we are concerned about performance. 
I saw the doc where the extractions' speed can be improved by using _repositoryCache_. 
Is this doable? As we run it the action as a pod in kubernetes, I guess the cache is always empty when running renovate.
Please advise.

### What you tried so far.

n/a

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
you need to store the cache on a persistent volume or use S3 as cache backend. also configure redis for package cache

**Reply by simone-dandreta** [reply]
what exactly will be cached?

**Reply by rarkins** [reply]
Extracted dependencies, open branches, etc
